src/coreclr/vm/arm64/asmhelpers.asm

```diff
@@ -676,6 +676,13 @@ COMToCLRDispatchHelper_RegSetup
 
 #endif ; FEATURE_HIJACK
 
+; void* PacStripPtr(void *);
+    LEAF_ENTRY PacStripPtr
+        ; xpaci    x0
+        DCD     0xDAC143E0
```

Comment:
`0xDAC143E0` <- Is that opcode? can you write a comment about it?

Reply:
This is the binary version of the `xpaci    x0` instruction. I'll elaborate the comment above.

Reply:
Put some comments about using the binary version as well using the `xpaclri` instead of `xpaci`.